docs/features/mathtype/README.md

```diff
@@ -137,7 +137,7 @@ If you visit a page using MathType with your mobile device, the handwriting inte
 
 ## Input and Output
 
-MathType is based upon standards like MathML for internal representation and the PNG image format for displaying formulas. It can also handle other formats like LaTeX, Flash, SVG and EPS.
+MathType is based upon standards like MathML for internal representation and the PNG image format for displaying formulas. It can also handle other formats like LaTeX, SVG and EPS.
```

Comment:
```suggestion
MathType is based upon standards like MathML for internal representation and the PNG image format for displaying formulas. It can also handle other formats like LaTeX, Flash, SVG and EPS.
```